typescript/web/src/components/invitation-manager/invitation-container.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Center,
+  Flex,
+  Heading,
+  useColorModeValue as mode,
+  VStack,
+} from "@chakra-ui/react";
+import { PropsWithChildren } from "react";
+
+type InvitationContainerProps = {
+  header: string;
+  details: string;
+};
+
+type CompleteProps = PropsWithChildren<InvitationContainerProps>;
+
+export const InvitationContainer = ({
+  header,
+  details,
+  children,
+}: CompleteProps) => {
+  return (
+    <Center h="full">
+      <VStack
+        p={6}
+        spacing={6}
+        align="start"
+        bg={mode("white", "gray.800")}
+        borderRadius="md"
+        boxShadow="lg"
+        maxW="2xl"
+        my="3.75rem"
+      >
+        <Heading fontSize="xl" fontWeight="semibold">
```

Comment:
Why don't you use the as property with an appropriate value?

Reply:
I should've explained it to you last time you made that comment.

`as` doesn't do anything on the style, it's just to include the right tag in the DOM. You can put whatever value you want in `as` it won't change the size.